pkgs/development/python-modules/debts/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, python, buildPythonPackage, fetchFromGitLab, jinja2 }:
+
+buildPythonPackage rec {
+  pname = "debts";
+  version = "0.5";
+
+  # pypi does not ship tests
+  src = fetchFromGitLab {
+    domain = "framagit.org";
+    owner = "almet";
+    repo = "debts";
+    rev = "09ee167c"; # no tags
+    sha256 = "1a3smld9x0xjppim9jgxp0zi2j6r8919nfs2lp8d9cfkq0kvzlmm";
+  };
+
+  propagatedBuildInputs = [ jinja2 ];
+
+  # for some reason tests only work if the module is properly installed
+  doCheck = false;
+  postInstall = ''
+    rm -r debts
+    export PYTHONPATH=$out/${python.sitePackages}:$PYTHONPATH
+    python nix_run_setup test
```

Comment:
Why `postInstall`? Seems like this should be `checkPhase`. Note that the python `checkPhase` is actually `installCheckPhase`, so its always run after installation.

Reply:
Oh I didn't know checkPhase was after installPhase for python packages. I reworked this.